src/scenic/syntax/translator.py

```diff
@@ -985,19 +1029,32 @@ def wrapStatementCall():
 					inConstructorContext = True
 					allowedPrefixOps = self.specifiersForConstructor(context)
 				else:
-					for opTokens, op in infixTokens.items():
-						if not op.contexts or context in op.contexts:
-							allowedInfixOps[opTokens] = op.token
 					for name, mod in modifierNames.items():
 						if not mod.contexts or context in mod.contexts:
 							allowedModifiers[name] = mod.name
 					if context in modifierNames:
 						allowedTerminators = modifierNames[context].terminators
 					elif context in terminatorsForStatements:
 						allowedTerminators = terminatorsForStatements[context]
+			# construct allowedInfixOps
+			if context:
+				# all function calls on the stack
+				contexts = set(ctx for ctx, _ in functionStack)
+				for opTokens, op in infixTokens.items():
+					# if not context is specified, the operator can be used
```

Comment:
"not" -> "no"

Reply:
Fixed 39474008e0421aa629590c8142ee0be642314e61